weddineo/src/main/resources/application.yml

```diff
@@ -1,10 +1,25 @@
 spring:
   profiles:
     active: '@spring.profiles.active@'
+  datasource:
+    url: jdbc:postgresql://192.168.99.100:5432/weddineo
+    username: postgres
```

Comment:
should we store this in here?

Reply:
Yes, until this properties are for developnent usage datasource config could be stored here in plain text(it refers to local database) - in future we could think about externalizing properties and attaching them during application launch.

I will replace IP in datasource.url to (...)localhost:5432(...) <- it was my mistake to commit sppecified addres :)